applications/serial_lte_modem/doc/MQTT_AT_commands.rst

```diff
@@ -194,16 +161,37 @@ Response syntax
 
 ::
 
-   #XMQTTEVT: <evt_type>,<result>
+   #XMQTTEVT=<evt_type>,<result>
 
-* The ``<evt_type>`` value is an integer.
-  When ``0``, it indicates the acknowledgment of the connection request.
-* The ``<result>`` value is an integer.
-  It can return the following values:
+* The ``<evt_type>`` value is an integer indicating the type of the event.
+  It can return the following values for the ``#XMQTTCON`` command:
+
+  * ``0`` - Acknowledgment of connection request (CONNACK).
+  * ``1`` - Disconnection notification (DISCONNECT).
+    The MQTT client is disconnected from the MQTT broker once this event is notified.
+
+* The ``<result>`` value is an integer. It can return the following values:
 
-  * ``0`` - Connection succeeded.
-  * *Negative Value* - Error code.
-    It indicates the reason for the failure.
+  * ``0`` - Success.
+  * *Negative value* - Error code indicating the reason for the failure.
+
+
+Unsolicited notification
+------------------------
+
+::
+   #XMQTTEVT=<evt_type>,<result>
+
+* The ``<evt_type>`` value is an integer indicating the type of the event.
+  It can return the following values after the connection is established:
+
+  * ``9`` - Ping response from the MQTT broker (PINGRESP).
+    This is received when we ping (PINGREQ) the broker after the keep alive is reached.
+
```

Comment:
```suggestion
* The ``<evt_type>`` value is an integer indicating the type of the event.
  After the connection is established, it can return ``9`` to indicate a ping response from the MQTT broker (PINGRESP).
  This is received when pinging (PINGREQ) the broker after the keep alive is reached.
```